docs: rearrange docs structure and prepare for future outline (#2051)
namehash/ensnode: docs/ensnode.io/src/content/docs/docs/reference/index.mdx

---
title: Reference

> description: Terminology, APIs, compatibility, and deeper ENSNode topics.

Reviewer: ```suggestion
description: Go deeper into mastery of ENSNode.
```

sidebar:
  label: Overview
  order: 1
---

import { LinkCard, CardGrid } from "@astrojs/starlight/components";

> Browse terminology, the roadmap, REST and legacy Subgraph surfaces, and contributing guides.

Reviewer: ```suggestion
Go deeper into mastery of ENSNode, including terminology and becoming an active contributor to ENSNode development.
```

<CardGrid>

>   <LinkCard title="Terminology" href="/docs/reference/terminology" description="ENSNode and multichain vocabulary used across the docs." />

Reviewer: ```suggestion
  <LinkCard title="Terminology" href="/docs/reference/terminology" description="Terminology for ENS development used across these docs." />
```

>   <LinkCard title="REST API" href="/docs/reference/rest-api" description="OpenAPI-backed HTTP API reference." />

Reviewer: ```suggestion
  <LinkCard title="REST API" href="/docs/reference/rest-api" description="ENSNode has additional APIs beyond the ENS Omnigraph and Subgraph. Explore additional OpenAPI-backed HTTP APIs." />
```

>   <LinkCard title="Subgraph compatibility" href="/docs/reference/subgraph-compatibility-tooling" description="Legacy subgraph-compatible GraphQL and migration context." />

Reviewer: ```suggestion
  <LinkCard title="Subgraph compatibility" href="/docs/reference/subgraph-compatibility-tooling" description="Tooling that guarantees the full backwards-compatibility of our (now deprecated!) ENS Subgraph API implementation." />
```

>   <LinkCard title="Contributing" href="/docs/reference/contributing" description="How to contribute to ENSNode and related projects." />

Reviewer: ```suggestion
  <LinkCard title="Contributing" href="/docs/reference/contributing" description="Learn how to contribute to the development of ENSNode and related projects." />
```
